conf.py

```diff
@@ -76,7 +76,7 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', "meta/README.rst"]
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', "meta/README.rst", ".venv"]
```

Comment:
I added the `.venv` here because following this steps in the root folder without put this config here:

Commands:
```sh
python3 -m venv .venv
make docs
```

Output:
```sh
Warning, treated as error:
testcontainers-python/.venv/lib/python3.11/site-packages/sphinx/ext/autosummary/templates/autosummary/base.rst:3:Error in "currentmodule" directive:
maximum 1 argument(s) allowed, 3 supplied.

.. currentmodule:: {{ module }}
make: *** [docs] Error 2
```